### ISSUE TITLE
Remove window resize event

### ISSUE THREAD

**Issue by Maxigong** [catdad/canvas-confetti#195]
Hello!
I was wondering if there is a way of removing the windows event it create, because the confetti resize events remain there even after user navigate to another route.

![Screenshot 2023-02-24 at 5 52 27 PM](https://user-images.githubusercontent.com/88868441/221135442-08274e2b-2acf-4edf-8cc7-c5f29332e663.png)

I tried the CDN link and the npm package and the situation is the same.

I am using this code:

```
const count = 100
    const defaults = {
        origin: { y: 0.35 },
    }
    function fire(particleRatio, opts) {
        confetti(
            Object.assign({}, defaults, opts, {
                particleCount: Math.floor(count * particleRatio),
            })
        )
    }

    fire(0.25, {
        spread: 26,
        startVelocity: 55,
    })
    fire(0.2, {
        spread: 60,
    })
    fire(0.35, {
        spread: 100,
        decay: 0.91,
        scalar: 0.8,
    })
    fire(0.1, {
        spread: 120,
        startVelocity: 25,
        decay: 0.92,
        scalar: 1.2,
    })
    fire(0.1, {
        spread: 120,
        startVelocity: 45,
    })

    setTimeout(() => {
        confetti.reset()
    }, 2000)

```

**Comment by catdad** [comment]
This is a bug. The event is supposed to be removed. Currently, if confetti is only fired once, the event will be removed. However, if fired multiple times without waiting for the previous to finish, an event is registered each time but only the last one is removed. This bug should be fixed.

**Comment by catdad** [comment]
A fix has been released in 1.6.1
